src/chartjs-plugin-trendline.js

```diff
@@ -49,9 +49,11 @@ function addFitter(datasetMeta, ctx, dataset, xScale, yScale) {
     if ( dataset.data && typeof dataset.data[0] === 'object') xy = true;
 
     dataset.data.forEach(function(data, index) {
-        if(data == null)
-            return;
-        if ( xy ) fitter.add(data.x, data.y);
+        if (data == null) return;
+
+        if (xScale.options.type === "time")
+            fitter.add(new Date(data.x).getTime(), data.y);
```

Comment:
`x` can also be `t` according to the [docs](https://www.chartjs.org/docs/latest/axes/cartesian/time.html)

Reply:
```suggestion
            fitter.add(new Date(data.x || data.t).getTime(), data.y);
```

Would do the trick ?

Reply:
doh dang I thought I PR'd into your fork https://github.com/Makanz/chartjs-plugin-trendline/pull/35